master/internal/trial.go

```diff
@@ -698,7 +700,7 @@ func (t *trial) transition(s model.StateWithReason) error {
 	if t.state != s.State {
 		t.syslog.Infof("trial changed from state %s to %s", t.state, s.State)
 		if t.idSet {
-			if err := t.db.UpdateTrial(t.id, s.State); err != nil {
+			if err := db.UpdateTrial(context.Background(), t.id, s.State); err != nil {
```

Comment:
use `context.TODO()` in spots like this.

Reply:
ah okay, I think I got it backwards in terms of always using context.Background() & context.TODO()